cmd/aws-janitor-boskos/main.go

```diff
@@ -90,6 +92,7 @@ func init() {
 		"Resources with any of these tags will not be managed by the janitor. Given as a comma-separated list of tags in key[=value] format; excluding the value will match any tag with that key. Keys can be repeated.")
 	flag.Var(&includeTags, "include-tags",
 		"Resources must include all of these tags in order to be managed by the janitor. Given as a comma-separated list of tags in key[=value] format; excluding the value will match any tag with that key. Keys can be repeated.")
+	flag.Var(&skipResourceRecordSetTypes, "skip-resource-record-set-types", "A list of resource record types which should not be deleted, splitted using comma.")
```

Comment:
I think we have to initialize default value for the flag / var to `NS, SOA`, otherwise `--skip-route53-management-check` would not work by default without explicit `--skip-resource-record-set-types`.

Reply:
We can't set default value at this line, the reason is that flag.Var only supports:
flag.Var(&flagVal, "name", "help message for flagname").
Nor can we set default value when initializing the var: skipResourceRecordSetTypes, the reason is that for 
common.CommaSeparatedStrings, if it initialized with non-empty value, the "Set" method will return error once being called.
https://github.com/kubernetes-sigs/boskos/blob/3e30fa430bc9bd77ea67d3a70bc74ce43b00190c/common/common.go#L211

What we can do is that before converting the string array to set, we can set its value if its empty.
See line: 193